docs/book/v3/migration/v2-to-v3.md

```diff
@@ -0,0 +1,23 @@
+# Migration from Version 2 to 3
+
+laminas-filter version 3 makes a number of changes that may affect your application.
+This document details those changes, and provides suggestions on how to update your application to work with version 3.
+
+## Removed Filters & Features
+
+These filters were deprecated in the 2.0.x series of releases and have now been removed:
+
+- `Laminas\Filter\File\Decrypt`
+- `Laminas\Filter\File\Encrypt`
+- `Laminas\Filter\Blacklist`
+- `Laminas\Filter\Decrypt`
+- `Laminas\Filter\Encrypt`
+- `Laminas\Filter\StaticFilter`
+- `Laminas\Filter\Whitelist`
```

Comment:
I miss some informations what the replacement is or what the user can do instead.
For example:

- Blacklist – [DenyList](https://docs.laminas.dev/laminas-filter/standard-filters/#denylist)
- Whitelist – [AllowList](https://docs.laminas.dev/laminas-filter/standard-filters/#allowlist)

```php
echo StaticFilter::execute('&', 'HtmlEntities');
```
```php
echo (new Laminas\Filter\HtmlEntities())->filter('&');
// or 
echo (new Laminas\Filter\HtmlEntities())('&');
```
https://docs.laminas.dev/laminas-filter/intro/#basic-usage-of-filters

and so on…